src/toil_scripts/adam_gatk_pipeline/align_and_call.py

```diff
@@ -117,132 +117,30 @@
 # import boto sdb connection for autoscaling
 import boto.sdb
 
+import textwrap
```

Comment:
I like lexicographic sort by module name. In a module, I then sort lexicographically again.